pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<groupId>org.springframework.boot</groupId>
 		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>2.2.5.RELEASE</version>
+		<version>2.1.5.RELEASE</version>
```

Comment:
why downgrade?

Reply:
I changed the version because was a problem to start the application.